frontend/src/components/RationalityPanel.svelte

```diff
@@ -0,0 +1,218 @@
+<script lang="ts">
+    import api from "$lib/api";
+    import Speedometer from "svelte-speedometer";
+    import type { Article } from "$lib/api/news";
+    import type { Trade } from "$lib/api/trades";
+    import { convertUnixDate } from "$lib/functions";
+    import { browser } from "$app/environment";
+    import NewsTileStatic from "./NewsTileStatic.svelte";
+    import ArcGauge from "./ArcGauge.svelte";
+
+    let active = false;
+    export let data: Trade;
+
+    // values should be somewhat normalized - i.e. it should be guaranteed that they fall within a constant range
+    const controversy = 300;
+    const risk = 500;
+    const evidence = 70;
+
+    let responses: Article[] = [];
+    const newNewsRequest = () => {
+        api.news()
+            .articles(data.article_evidence)
+            .then((response) => (responses = response));
+    };
+
+    $: if (browser) newNewsRequest();
+</script>
+
+{#if active}
+    <div class="modal is-active">
+        <!-- svelte-ignore a11y-click-events-have-key-events -->
+        <div class="modal-background" on:click={() => (active = false)} />
+        <div class="modal-content">
+            <div style="display: flex; justify-content: right;">
+                <button
+                    class="button is-ghost"
+                    on:click={() => (active = false)}>
+                    <span class="icon is-large">
+                        <i class="fas fa-xmark" />
+                    </span>
+                </button>
+            </div>
+
+            <div class="block mx-5">
+                <header class="title is-3">Feedback</header>
+                <hr style="background: #4a4a4a; height: 1px" />
+            </div>
+
+            <div class="block mx-5">
+                <header class="title is-5">Scores</header>
+
+                <div class="columns" style="height: 55vh">
+                    <div class="column">
+                        <!-- workaround for weird spacing of the speedometer -->
+                        <div style="height: 10vh" />
+                        <div
+                            style="height: 15vh; display: flex; justify-content: center; align-items: center;">
+                            <Speedometer
+                                value={controversy}
+                                segments={1000}
+                                maxSegmentLabels={0}
+                                needleColor="#f5f5f5"
+                                needleTransitionDuration={3000}
+                                needleTransition="easeElastic"
+                                needleHeightRatio={0.7}
+                                startColor="#e91e63"
```

Comment:
Maybe flip these colours round? I think blue normally has a better connotation than red (at least in western countries).

Reply:
Ah yes
Ur absolutely right I accidentally got this the other way around ;-;